mountpoint-s3/src/cli.rs

```diff
@@ -553,19 +551,30 @@ impl CliArgs {
         cache_config
     }
 
-    fn cache_express_bucket_name(&self) -> Option<&str> {
+    fn cache_express_bucket_name(&self) -> Option<Result<S3Path, S3PathError>> {
         if let Some(bucket_name) = &self.cache_xz {
-            return Some(bucket_name);
+            let s3_path = S3Path::new(bucket_name.to_owned(), Prefix::new("").unwrap());
```

Comment:
what's the point of allowing s3 path in `--cache-xz` argument? specifying prefix is not allowed there, is it just for uniformity?